src/Framework/Framework/Resources/Scripts/utils/evaluator.ts

```diff
@@ -33,6 +34,44 @@ export function traverseContext(context: any, path: string): any {
     return currentLevel
 }
 
+export function findPathToChildObject(vm: any, child: any, path: string): string | null {
+    if (vm == child) {
+        // We found the child
+        return path;
+    }
+
+    vm = ko.unwrap(vm);
+    if (typeof vm !== "object" || vm == null) {
+        return null;
+    }
+
+    if (Array.isArray(vm)) {
+        // Iterate over its elements
+        vm.forEach(function (value, index) {
+            let result = findPathToChildObject(value, child, path + "/" + `[${index}]`)
+            if (result != null)
+                return result;
```

Comment:
This will not work, it returns from the inner anonymous function, not from the `findPathToChildObject` function. You need to use `for(const value of vm) {... }` and track the index manually